examples/git-receive-pack/main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"gopkg.in/src-d/go-git.v4/plumbing/transport"
+	"gopkg.in/src-d/go-git.v4/plumbing/transport/file"
+)
+
+func main() {
+	if len(os.Args) != 2 {
+		fmt.Fprintln(os.Stderr, "usage: git receive-pack <git-dir>")
+		os.Exit(129)
+	}
+
+	if err := file.DefaultServer.Serve(
```

Comment:
I have not used it here so that the example behaves exactly the same as the official git-receive-pack.